pkgs/top-level/all-packages.nix

```diff
@@ -14959,6 +14959,8 @@ with pkgs;
 
   schedtool = callPackage ../os-specific/linux/schedtool { };
 
+  scss_lint = callPackage ../development/tools/scss_lint { };
```

Comment:
You are in the `OS-SPECIFIC` section there.
But there is a `### DEVELOPMENT / TOOLS` section further up that would be more suited.

Reply:
Thanks. I'll fix.